prometheus_juju_exporter/config_default.yaml

```diff
@@ -13,9 +13,10 @@ exporter:
   collect_interval: 15
 
 detection: # parameters affecting the detection algorithm
-  virt_macs: 
+  skip_interfaces: []  # regexp list of interfaces that should be skipped during hw typ detection
```

Comment:
I think we should provide a default list of interfaces here, similar to the `virt_macs`. We cannot assume the snap will always be used with the charm, and these interfaces should be ignored in most (if not all) cases.

Reply:
After discussion, we agreed to keep the `skip_interface` config option empty in the snap config and let charm take charge of setting it.

For the `virt_macs` option, we will open another PR to remove its default value.